SPECS/xen.spec

```diff
@@ -80,6 +80,10 @@ Source51: xen-kernel.aarch64
 Source52: efi-xen.cfg.aarch64
 Source53: edk2-bc54e50e0fe03c570014f363b547426913e92449.tar.gz
 
+%if %{with_livepatch}
+Source60: livepatch-tools-2af6f1aa62334f8c3d66cf2c5043686833de6cc6.tar.gz
```

Comment:
I'll fix this on check-in; as discussed I think the other things we can fix up later.